og_migrate/migrations/d7_og_role.yml

```diff
@@ -0,0 +1,59 @@
+id: d7_og_role
+label: Og roles
+migration_tags:
+  - Drupal 7
+source:
+  plugin: d7_og_role
+process:
+  id:
+    -
+      plugin: concat
+      source:
+        - group_type
+        - group_bundle
+        - name
+      delimiter: '-'
+    -
+      plugin: og_role_machine_name
+    -
+      plugin: make_unique_entity_field
+      entity_type: og_role
+      field: id
+      migrated: true
+  label: name
+  weight: rid
+  # This could lead to bad data on the destination if something happened to the
+  # group entity_type, bundle or entity in the source or in the migration of
+  # those things to the destination. It's not possible to use migrate_lookup
+  # because the migration could be literally any migration that deals with an
+  # entity_type in the source.
+  group_type: group_type
+  group_bundle: group_bundle
+  group_id: gid
+  # @todo How can this be detected?
```

Comment:
@mradcliffe, I missed this the first time. Shouldn't we check here if name is "administrator member"?

Reply:
administrator-member is transformed to administrator